ert_gui/ertwidgets/analysismodulevariablespanel.py

```diff
@@ -175,8 +176,14 @@ def createSpinBox(
     ):
         spinner = QSpinBox()
         spinner.setMinimumWidth(75)
+
+        # setMaximum() expects a signed 32bit
+        max_int = int(math.pow(2, 31) - 1)
         spinner.setMaximum(
-            analysis_module_variables_model.getVariableMaximumValue(variable_name)
+            min(
```

Comment:
In stead of this value truncation, could it make sense to add a test that goes through all the _VARIABLE_NAMES, and calls getVariableMaximumValue() functions, and possibly other ones as well?

Reply:
We think so, because a test would give feedback as to why the number (or other values) needs to be of a certain range. While this truncation will just silently ignore illegal values.

Reply:
Good idea. I will look into adding test(s) for this